src/themes/common.tsx

```diff
@@ -8,7 +8,8 @@ export const dimensions = {
     weights: {
       light: 1,
       regular: 1.5,
-      heavy: 2
+      heavy: 2,
+      strong: 3,
```

Comment:
The weights I have in Sketch are 1, 1.5, 3 and 5 so we definitely need an extra level but it those heavier weights are rarely used so if we're needing them in this feature we need to look at if there's a problem in the icon workflow somewhere. 

Originally the icons should have been setup to scale in the browser in the same way that Sketch handles it. This is part of the reason we have a script to parse icons to add them to the kit. Does the weighting on this feature have the same issue if you choose different icons or is it just an issue with this icon?

Reply:
About the icon weights

<img width="1130" alt="Screen Shot 2021-08-18 at 17 10 42" src="https://user-images.githubusercontent.com/10409078/129862364-90bc53cb-1c84-4dab-b80e-f7fcc932c1ca.png">

Well most of them have this problem and some do not upscale that good 🙈 

Zone
<img width="191" alt="Screen Shot 2021-08-18 at 17 15 39" src="https://user-images.githubusercontent.com/10409078/129863068-0a4efb59-2d1f-4910-a172-559fb4047132.png">

Retry Job
<img width="125" alt="Screen Shot 2021-08-18 at 17 17 02" src="https://user-images.githubusercontent.com/10409078/129863246-4d0bd094-0360-4b60-b295-b2325864fe40.png">

Temperature
<img width="135" alt="Screen Shot 2021-08-18 at 17 17 24" src="https://user-images.githubusercontent.com/10409078/129863296-7b9a4726-96af-4cdf-b3ed-aa8d005028d6.png">

Usage
<img width="142" alt="Screen Shot 2021-08-18 at 17 18 16" src="https://user-images.githubusercontent.com/10409078/129863407-0f5e1d84-9687-4f74-baeb-ef201ca9b822.png">

Critical
<img width="216" alt="Screen Shot 2021-08-18 at 17 14 32" src="https://user-images.githubusercontent.com/10409078/129862914-ba3c6459-9741-4db1-8b11-cf5f2d1fe9a3.png">

Probably we shouldn't use this weight for now..